lib/manager/util/index.js

```diff
@@ -397,3 +421,10 @@ export function projectHasAtLeastOneFeedWithAPublishedVersion (project: ?Project
 export function validationState (val: ?boolean): ?string {
   return val ? undefined : 'error'
 }
+
+export function stringifyLabels (labels: Array<any>): Array<any> {
+  // Convert feed source labels to id strings wanted by datatools-server,
+  // but only if we have a label object
+  if (!labels) return []
+  return labels.map(label => label.id ? label.id : label)
+}
```

Comment:
Please remove if refactoring data model as recommended in https://github.com/ibi-group/datatools-server/pull/403#pullrequestreview-714190079.